### PR TITLE
Add more advanced styles

### PR DESCRIPTION
**Pull request by laptou**
Add more advanced styles to make the page look prettier, including changing the font families and highlight colours to be consistent w/ my resume, and giving the food sections a more interesting layout.

image

This closes #5.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css
Comment (jpnickolas) [inline portfolio/src/main/webapp/style.css]: nit: comment what color these are since hsl (and honestly even hex colors that aren't white or black) are hard to read.

**Comment by jpnickolas** [comment]
LGTM. Nice use of grid styles. Will approve once parent PR's are merged.

**Review by jessicaslaughter — APPROVED** [review]

**Review by jpnickolas — CHANGES_REQUESTED** [review]

**Review by jpnickolas — APPROVED** [review]

**Pull request merged**
